### DISCUSSION TITLE
Labelling work for cell and item detection

### DISCUSSION THREAD

**Discussion by anson416** [deepdoctection/deepdoctection#188] [Q&A] [answered]
I am trying to finetune the given cell and item detection models with my own dataset. I see that you have your own models on your Hugging Face Space. How did you label your data? What concerns me the most are row spanning and column spanning.

**Comment by JaMe76** [comment]
Hi,

I labeled around 1 k table cells, rows and columns using Prodigy.

To label not everything from scratch I added model proposals that I reviewed. 

To be very clear: It is a pain and I haven’t found any better solution. For my use-cases it improved a lot though.

**Reply by anson416** [reply]
But how do you handle row span and column span? For example:
```text
-------
|     |
-------
|  |  |
-------
```
Do you still label columns as:
```text
-------
|  |  |
|--|--|
|  |  |
-------
```
?

**Reply by JaMe76** [reply, marked as answer]
I label rows and columns as in your second sample. As cells are labeled independently, they can overlap with multiple rows/cols. Once they exceed an overlapping threshold, they span over the row.

Rows and columns in this approach are only there to create a grid of the tables.

**Reply by anson416** [reply]
Thanks!
